ccmlib/scylla_node.py

```diff
@@ -715,8 +716,20 @@ def import_bin_files(self):
                 self._launch_env['LD_LIBRARY_PATH'] = dbuild_so_dir
 
                 if scylla_mode == 'debug':
-                    cmd = ['strip', dst]
-                    subprocess.check_call(cmd)
+                    cmd = [loader, os.path.join(dbuild_so_dir, 'patchelf'), '--version', loader]
+                    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=self._launch_env)
+                    (stdout, stderr) = p.communicate()
+                    if p.returncode != 0:
+                        raise RuntimeError('{}: exited with status {}.\nstdout:{}\nstderr:\n{}'.format(cmd, p.returncode, stdout, stderr))
+                    match = re.match('^patchelf (\d)+\.(\d+)$', stdout)
+                    if match:
+                        major = int(match.group(1))
+                        minor = int(match.group(2))
+                        if major <= 0 and minor < 10:
+                            cmd = ['strip', dst]
+                            subprocess.check_call(cmd)
+                    else:
+                        print('Could not determine patchelf version: {}'.format(stdout))
```

Comment:
exit?

Reply:
The idea is to attempt to run patchelf any case and let it fail, need be.